godotproject/Menus/InGameMenu/InGameMenu.tscn

```diff
@@ -135,7 +135,6 @@ tracks/2/keys = {
 [node name="InGameMenuLayer" type="CanvasLayer"]
 
 [node name="InGameMenu" type="MarginContainer" parent="."]
-pause_mode = 2
```

Comment:
We want to avoid setting "PAUSE_MODE_PROCESS" wherever possible so that the scope of "what's running" vs. "what's not" is clear.